config.fnl

```diff
@@ -326,22 +345,24 @@
 ;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
 
 (local menu-items
-       [{:key :space
-         :title "Alfred"
+       [{:key    :space
+         :title  "Alfred"
          :action (activator "Alfred 4")}
-        {:key :w
+        {:key   :w
          :title "Window"
+         :enter (fn [] (windows.show-display-numbers))
+         :exit  (fn [] (windows.hide-display-numbers))
```

Comment:
The first argument is also the window-menu table so I think this would be a good place to remove all the monitor bindings so that if the number of monitors changes while spacehammer is running, we are not pointing to phantom monitors.